### ISSUE TITLE
training flag in dropout layer

### ISSUE THREAD

**Issue by Turtle-Duck** [ly-atdawn/Unet-Tensorflow#4]
The training flag used in dropout layer seems doesn‘t be changed in test stage?
